<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserViewer</class>
 <widget class="QWidget" name="UserViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>897</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User Viewer</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="group_budgets">
     <property name="title">
      <string>Budgets</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QListWidget" name="budgets_list">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="budgets_buttons">
        <item>
         <widget class="QPushButton" name="add_budget">
          <property name="text">
           <string> Add</string>
          </property>
          <property name="icon">
           <iconset resource="../../picsou.qrc">
            <normaloff>:/resources/material-design/svg/book-plus.svg</normaloff>:/resources/material-design/svg/book-plus.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="edit_budget">
          <property name="text">
           <string>Edit</string>
          </property>
          <property name="icon">
           <iconset resource="../../picsou.qrc">
            <normaloff>:/resources/material-design/svg/book-open.svg</normaloff>:/resources/material-design/svg/book-open.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="remove_budget">
          <property name="text">
           <string>Remove</string>
          </property>
          <property name="icon">
           <iconset resource="../../picsou.qrc">
            <normaloff>:/resources/material-design/svg/book-minus.svg</normaloff>:/resources/material-design/svg/book-minus.svg</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="group_accounts">
     <property name="title">
      <string>Accounts</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QListWidget" name="accounts_list">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="accounts_buttons">
        <item>
         <widget class="QPushButton" name="add_account">
          <property name="text">
           <string>Add</string>
          </property>
          <property name="icon">
           <iconset resource="../../picsou.qrc">
            <normaloff>:/resources/material-design/svg/playlist-plus.svg</normaloff>:/resources/material-design/svg/playlist-plus.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="edit_account">
          <property name="text">
           <string>Edit</string>
          </property>
          <property name="icon">
           <iconset resource="../../picsou.qrc">
            <normaloff>:/resources/material-design/svg/pencil.svg</normaloff>:/resources/material-design/svg/pencil.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="remove_account">
          <property name="text">
           <string>Remove</string>
          </property>
          <property name="icon">
           <iconset resource="../../picsou.qrc">
            <normaloff>:/resources/material-design/svg/playlist-remove.svg</normaloff>:/resources/material-design/svg/playlist-remove.svg</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="transfer">
        <property name="text">
         <string>Transfer</string>
        </property>
        <property name="icon">
         <iconset resource="../../picsou.qrc">
          <normaloff>:/resources/material-design/svg/swap-horizontal.svg</normaloff>:/resources/material-design/svg/swap-horizontal.svg</iconset>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="action_add_budget">
   <property name="icon">
    <iconset resource="../../picsou.qrc">
     <normaloff>:/resources/material-design/svg/book-plus.svg</normaloff>:/resources/material-design/svg/book-plus.svg</iconset>
   </property>
   <property name="text">
    <string>Add budget</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+B</string>
   </property>
  </action>
  <action name="action_edit_budget">
   <property name="icon">
    <iconset resource="../../picsou.qrc">
     <normaloff>:/resources/material-design/svg/book-open.svg</normaloff>:/resources/material-design/svg/book-open.svg</iconset>
   </property>
   <property name="text">
    <string>Edit budget</string>
   </property>
  </action>
  <action name="action_remove_budget">
   <property name="icon">
    <iconset resource="../../picsou.qrc">
     <normaloff>:/resources/material-design/svg/book-minus.svg</normaloff>:/resources/material-design/svg/book-minus.svg</iconset>
   </property>
   <property name="text">
    <string>Remove budget</string>
   </property>
  </action>
  <action name="action_add_account">
   <property name="icon">
    <iconset resource="../../picsou.qrc">
     <normaloff>:/resources/material-design/svg/playlist-plus.svg</normaloff>:/resources/material-design/svg/playlist-plus.svg</iconset>
   </property>
   <property name="text">
    <string>Add account</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+A</string>
   </property>
  </action>
  <action name="action_edit_account">
   <property name="icon">
    <iconset resource="../../picsou.qrc">
     <normaloff>:/resources/material-design/svg/pencil.svg</normaloff>:/resources/material-design/svg/pencil.svg</iconset>
   </property>
   <property name="text">
    <string>Edit account</string>
   </property>
  </action>
  <action name="action_remove_account">
   <property name="icon">
    <iconset resource="../../picsou.qrc">
     <normaloff>:/resources/material-design/svg/playlist-remove.svg</normaloff>:/resources/material-design/svg/playlist-remove.svg</iconset>
   </property>
   <property name="text">
    <string>Remove account</string>
   </property>
  </action>
  <action name="action_transfer">
   <property name="icon">
    <iconset resource="../../picsou.qrc">
     <normaloff>:/resources/material-design/svg/swap-horizontal.svg</normaloff>:/resources/material-design/svg/swap-horizontal.svg</iconset>
   </property>
   <property name="text">
    <string>Transfer</string>
   </property>
   <property name="toolTip">
    <string>Transfer money between accounts</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+T</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../picsou.qrc"/>
 </resources>
 <connections/>
</ui>
